### PR TITLE
Start edit pregnancy form

### PR DESCRIPTION
**Pull request by charleshuang80**
Started the edit pregnancy form with patient and clinic information. Will continue on the rest of the form.

### CODE REVIEW

File: app/models/pregnancy.rb [hunk 1]
```diff
@@ -3,39 +3,46 @@ class Pregnancy
 	include Mongoid::Timestamps
 	include Mongoid::History
 
-	field :voicemail_ok, type: Boolean
+	# relationships
```
Comment (colinxfleming) [inline app/models/pregnancy.rb]: Nailed it. Thanks a ton for this reorg!

File: app/views/pregnancies/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<!-- Patient Info -->
+<%= bootstrap_form_for @pregnancy do |preg| %>
```
Comment (colinxfleming) [inline app/views/pregnancies/edit.html.erb]: A++++ view writin'

**Comment by colinxfleming** [comment]
Looks like this fixes the following: 

#92 

Gets us most of the way there on: 

#95 
#94

**Comment by charleshuang80** [comment]
Also, sorry, but you should take out the `<% console %>` on line 56 if you want to see how it will look or for tests (it is for while I've been working on it).

**Pull request merged**